ApplicationManager.js

```diff
@@ -33,12 +34,23 @@ class ApplicationManager {
     this.paramButtons = {};
     this.resetParametersButtons();
 
+    this.xiniclick = parameterInputFactory(
+      null,
+      inputTypes.click,
+      null,
+      null,
+      xini => this.setParam(paramNames.x_ini, xini),
+      "#svg1"
+    );
+
     this.plotFunctionAndAxis();
     this.plotAlgoResults();
   }
 
-  resetParams(algoName) {
-    this.algoParams = algorithmsConfig[algoName].parameters.reduce(
+
+
+  get_Initialized_Params(algoName) {
```

Comment:
A small comment on the naming convention for class methods:
```suggestion
  getInitializedParams(algoName) {
```